tests/testthat/test-copy-model-from.R

```diff
@@ -153,7 +153,8 @@ test_that("copy_from_model .overwrite=FALSE works [BBR-CMF-005]", {
 })
 
 test_that("copy_model_from() supports `.new_model` containing a period [BBR-CMF-006]", {
-  temp_mod_path <- create_temp_model()
+  mod_content <- readLines(ctl_ext(MOD1_PATH)) %>% paste(collapse = "\n")
+  temp_mod_path <- create_temp_model(mod_content = mod_content)
```

Comment:
The need for these test adjustments makes sense.

The test above already covers the main `.update_model_file = TRUE` case, but it'd be nice to have a test case that covered something that would have failed before this transition (e.g., a lower case `$prob ...`). It'd also be good to test the bare `$PROB{newline}` case mentioned above.

Reply:
oh just noticed the lower case `$prob` test case. Will add that now before you review

Reply:
Ok added that case in [789b9fe](https://github.com/metrumresearchgroup/bbr/pull/604/commits/789b9fefbf06b7a93987df6e48f67478f5f9ffea)